packages/sqlite/package.json

```diff
@@ -32,7 +32,7 @@
   "dependencies": {
     "@keyvhq/sql": "^1.6.6",
     "pify": "~5.0.0",
-    "sqlite3": "~5.0.2"
+    "sqlite3": "npm:@vscode/sqlite3@~5.0.7"
```

Comment:
is this way of declaring the dependency actually a necessary thing?

why not `"@vscode/sqlite3": "~5.0.7"` ?

Reply:
eh it wouldnt' matter we just need to change it to @vscode/sqlite3 in the code